lesson03/assignment/css/style.scss

```diff
@@ -4,3 +4,121 @@
 @import 'settings.responsive';
 @import 'components.content';
 
+// responsive sizing for "screen and (max-width: 767px)"
+@include media-query('mobile') {
+  // border styling
+  .template-hook {
+    display: block;
+    border: 3rem solid $darkgray;
+    border-radius: 10px;
+    //margin: 1rem;
+    color: $darkergray;
+  }
+  // all card info
+  .card {
+    text-align: center;
+    img {
+      border-radius: 5%;
+      width: 50%;
+    }
+    h1 {
+      color: $darkergray;
+      margin-bottom: 0;
+    }
+    p {
+      font-style: italic;
+      font-size: 23px;
+      margin-top: 0;
+    }
+  }
+  .card-more {
+    background-color: $gray;
+    padding: 1rem 2rem;
+    font-size: 18px;
+    // hide the link underline
+    a:link {
+      text-decoration: none;
+    }
+    // show the link underline on hover
+    a:hover {
+      text-decoration: underline;
+    }
+  }
+}
+// responsive sizing for "screen and (min-width: 768px)"
+@include media-query('tab') {
+  // border styling
+  .template-hook {
+    display: flex;
+    border: 3rem solid $darkgray;
+    border-radius: 10px;
+    margin: 1rem;
+    color: $darkergray;
+  }
+  // card info
+  .card {
+    flex: 30%;
+    text-align: center;
+    img {
+      border-radius: 50%;
+      width: 100%;
```

Comment:
This would normally be ok, but you'll want to make sure the parent element of your img has a defined size so the img doesn't appear too large